spec/minimax_spec.rb

```diff
@@ -1,170 +1,111 @@
 require 'board'
-require 'minimax'
+require 'win_checker'
 require 'score_tracker'
+require 'minimax'
 
 WIN_SCORE = ScoreTracker::WIN_SCORE
 LOSS_SCORE = ScoreTracker::LOSS_SCORE
 DRAW_SCORE = ScoreTracker::DRAW_SCORE
 
 describe 'minimax_move_and_score' do
+  before(:each) do
+    @b = Board.new(3)
```

Comment:
One-letter variable names are usually a bad idea.  I'm ok with it when their scope is limited to one or two lines and what they represent is either obvious or abstract.

`@w` and `@s` should be local variables.  They aren't used anywhere else.  If they are local variables, I'm ok with them being one-letter because they are limited in scope and obvious.  `@b` and `@m` however are used in other places and should have better names.

Reply:
I edited the variable names. However, right now i'm focusing on the kata version, which is different than the idea Minimax version (so when I'm done with the kata, I'll add back in the constants, refactor to use the Hash, and merge the branch into master)